docs/xxjitserveraotcachedir.md

```diff
@@ -0,0 +1,51 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:JITServerAOTCacheDir
+
+**(Linux&reg; only)**
+
+This option specifies the directory to be used for saving and loading JITServer AOT cache files.
+
+## Syntax
+
+        -XX:JITServerAOTCacheDir=<directory>
+
+ If the directory name is not specified (`-XX:JITServerAOTCacheDir=`), the cache files are saved in the current working directory of the JITServer server.
```

Comment:
This gives the impression that `-XX:JITServerAOTCacheDir=` is a valid option, where in fact it's not. I would eliminate the expression in parenthesis. 
Or, we could say: "If the option is not specified, the cache files are saved in the current working directory of the JITServer server."